src/common/envalid.ts

```diff
@@ -15,11 +31,9 @@ const cleanSpec = {
   TRACE: bool({ default: false }),
   VALUES_INPUT: str({ desc: 'The chart values.yaml file', default: undefined }),
 }
-let pEnv: any = process.env
-const path = `${pEnv.ENV_DIR}/.secrets`
-if (pEnv.ENV_DIR && existsSync(path)) {
-  const result = config({ path }) // this sets vars from .env onto process.env
-  if (result.error) console.error(result.error)
-  pEnv = { ...pEnv, ...result.parsed }
+
+const path = `${process.env.ENV_DIR}/.secrets`
+if (process.env.ENV_DIR && existsSync(path)) {
```

Comment:
add comment that you do it because to introduce caching like mechanism

Reply:
It doesn't?

Reply:
It just checks if the files are present, otherwise skip.